issueWork/JS1__issue/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script defer>
+    const numArr = [15,52,33,24,75];
+    let counter = 0;
+    for(let i = 0; i < numArr.length; i++) {
+      numArr[i]+=counter;
+      counter++;
+    }
+    function consoleLoggingAnArray (arr) {
+      let i = 0;
+      while(i < arr.length) {
+
+      if(arr[i] % 2 === 0) {
+        i++;
+        console.log("The number is even.");
+      }
+
+      i++;
+      console.log("The number is odd.");
+    }
+  }
+    consoleLoggingAnArray(numArr);
+    consoleLoggingAnArray([42,36,27,55]);
+    numArr.splice(2, 1);
```

Comment:
Suggestion:

Similar to what was mentioned above, I recommend adding spaces to separate your `consoleLoggingAnArray` function calls and the numArr mutations.